### ISSUE TITLE
Make build time average build time mean

### ISSUE THREAD

**Issue by NobbZ** [maralorn/nix-output-monitor#56]
Currently it is documented that the previous bildtimes are averaged.

It would probably make sense to use a mean here, as thats more resistant against outliers.

**Comment by maralorn** [comment]
I agree that this could deliver more robust information. The advantage of the rolling average we are currently using is, that it prevents database blow-up because we only need to save one number per package.

**Comment by SuperSandro2000** [comment]
> thats more resistant against outliers.

I think we need something smarter than average build time. Sometimes (to often) nix has hicups where it suddenly takes 10 minutes to build a derivation which should take 1 minute. Also when building many things in parallel chromium can take over an hour where it usually takes 30 minutes.

**Comment by maralorn** [comment]
Will tackle this together with #104
